espnet2/asr/encoder/conformer_encoder.py

```diff
@@ -354,10 +356,19 @@ def forward(
                         else:
                             xs_pad = xs_pad + self.conditioning_layer(ctc_out)
 
-        if isinstance(xs_pad, tuple):
-            xs_pad = xs_pad[0]
-        if self.normalize_before:
-            xs_pad = self.after_norm(xs_pad)
+        # This was not found helpful but keep this for now
```

Comment:
Could you clean the code here?

Reply:
I agree.
If it is not helpful, we do not have to put it in the main repo.